src/LabelSelect/widget/ui/LabelSelect.css

```diff
@@ -583,7 +583,7 @@ a.ui-button:active,
   margin: 0;
   cursor: pointer;
   /* support: IE10, see #8844 */
-  list-style-image: url("data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7");
+  list-style-image: url(LabelSelect/widget/ui/images/empty-list-marker.gif);
```

Comment:
Is there a reason this url is relative to `src` and not relative to the file itself? So `./images/empty-list-marker.gif`

Reply:
Yes, webpack just copy CSS, without any processing, so this path will be in final CSS bundle, so this path is relative to widget bundle root